Application/EdFi.Common/Configuration/DatabaseEngine.cs

```diff
@@ -21,6 +21,17 @@ public DatabaseEngine(string value, string displayName, string scriptsFolderName
         }
 
         public string ScriptsFolderName { get; private set; }
+
+        public static DatabaseEngine TryParseEngine(string value)
```

Comment:
If we want an exception why not just use Parse?  Does it have to be this message and a NotSupportedException?

Reply:
Here I was just following the same pattern as the `ApiMode`